collectors/rest/salesforce/breaker.json

```diff
@@ -0,0 +1,26 @@
+{
+  "id": "breaker",
```

Comment:
Please use a more descriptive name of the breaker. E.g. `sfdc-rest-breaker`

Reply:
@Algi1 looks like only the breaker rule name was updated, not the break name itself. Can you fix this too please? `id: breaker`

Reply:
Missed that. This has now been corrected